README.md

```diff
@@ -69,7 +77,9 @@ project using a rpm package. This can be done following the next steps (this exa
 
 ```
 # All the command must be executed as root user
+# The hanadb_exporter uses the prometheus_client for python. You can find it on the ha-clustering:Factory Repository.
```

Comment:
is this not in the `monitoring` repo? perhaps we could add it there

Reply:
@MalloZup No, it is on the Devel:Languages:Python as it is a library, but the problem is that on the languages project, it only builds for the most recent versions.. not for SLE-12, for example.

Reply:
ok thx